### PR TITLE
Improved highlighting of Environment

### PR DESCRIPTION
**Pull request by philippe-leonhardt-cko**
Added environment info overlay on startup and navigation to other environments

### CODE REVIEW

File: CKODemoShop/ClientApp/src/app/services/app-config.service.ts [hunk 1]
```diff
@@ -1,5 +1,29 @@
 import { Injectable } from '@angular/core';
 import { Location } from '@angular/common';
+import { IWebPaymentDemoEnvironment } from '../interfaces/web-payment-demo-environment.interface';
+
+const WEBPAYMENTDEMOENVIRONMENTS: IWebPaymentDemoEnvironment[] = [
+  {
+    name: 'Development',
+    url: null,
```
Comment (mikhail-kuzmiankou-cko) [inline CKODemoShop/ClientApp/src/app/services/app-config.service.ts]: Wny is it null? Do we support `Dev` env?

Comment (mikhail-kuzmiankou-cko) [reply CKODemoShop/ClientApp/src/app/services/app-config.service.ts]: Is it for local development?

Comment (philippe-leonhardt-cko) [reply CKODemoShop/ClientApp/src/app/services/app-config.service.ts]: correct, it's local and you are not supposed to have a link to the Dev instance when on any other instance

Comment (mikhail-kuzmiankou-cko) [reply CKODemoShop/ClientApp/src/app/services/app-config.service.ts]: As clarified offline, after the website is switched to other env, it doesn't show a button for local env. So this url is never used in local env.

**Pull request merged**